entities/order.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'ffaker'
+require './tasks/validate'
+
+# orders of books
+class Order
+  include Validate
+  attr_reader :book, :reader, :date
+
+  def initialize(book, reader, date)
```

Comment:
https://apidock.com/ruby/Date/today/class
initialize(book, reader, date = Date.today)

Reply:
@VKuzmich not fixed

Reply:
done